### PR TITLE
Feat(admin-ui): show total items in datatables

### PR DESCRIPTION
**Pull request by martijnvdbrug**
Show current start,end and total items in datatable:

image

image

image

image

Not too comfortable with Angular, so let me know if this is the desired approach.

### CODE REVIEW

File: docs/content/developer-guide/deployment.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ and you should expect to see `UTC` or `Etc/UTC`.
 For a production Vendure server, there are a few security-related points to consider when deploying:
 
 * Set the [Superadmin credentials]({{< relref "auth-options" >}}#superadmincredentials) to something other than the default.
-* Disable introspection in the [ApiOptions]({{ relref "api-options" }}#introspection) (this option is available in v1.5+).
+* Disable introspection in the [ApiOptions]({{< relref "api-options" >}}#introspection) (this option is available in v1.5+).
```
Comment (martijnvdbrug) [inline docs/content/developer-guide/deployment.md:42]: Bugfix from master, leave it in here?

**Comment by martijnvdbrug** [comment]
Good point, done :) Translations for these were pretty easy, just one word :rofl:

File: docs/content/developer-guide/deployment.md (hunk 1 above)
Comment (michaelbromley) [reply docs/content/developer-guide/deployment.md:42]: Yes that's fine 👍

**Comment by michaelbromley** [comment]
Thank you!

**Pull request merged**